torch/_dynamo/variables/torch.py

```diff
@@ -367,6 +377,25 @@ def get_state_from_generator():
                 ),
                 **options,
             )
+        elif all([isinstance(x, DynamicShapeVariable) for x in args]):
```

Comment:
This if statement is weirdly asymmetric with the rest of the if statements, which are querying against self.value.  Instead, this statement shunts you into this pass if all of the arguments are dynamic shape variables, no matter what. Indeed, it seems like you've just duplicated the behavior below otherwise. Maybe it would make sense to just have a small thing that shunts from sqrt to sym_sqrt...

Reply:
I did this for readability, as rewriting the `fn_` in call_function is not a common pattern. I can collapse the cases if you find that more readable.